spec/index.bs

```diff
@@ -980,6 +980,34 @@ The default is used if a value is not explicitly specified in {{GPUDeviceDescrip
         <tr><th>Limit name <th>Type <th>[=limit/Better=] <th>[=limit/Default=]
     </thead>
 
+    <tr><td><dfn>maxTextureDimension1D</dfn>
+        <td>{{GPUSize32}} <td>Higher <td>4096
+    <tr class=row-continuation><td colspan=4>
+        The maximum allowed {{GPUTextureDescriptor/size}}.{{GPUExtent3DDict/width}}
+        for which the {{GPUTextureDescriptor/dimension}} is {{GPUTextureDimension/"1d"}}
```

Comment:
"for which" confuses me a great deal here. Could this be reworded? Something like "for textures created with ..."

Reply:
"for textures created with ... is ..." seems not good. And It is tailed with "when creating a GPUTexture", should I remove this clause because it is duplicated? 

BTW, Descriptions for other parameters is using "for which ... is ... when ... ".  I removed "the" though. Or maybe we can say "for whose ... is ... when ..."? I am not a native English speaker. I am really open-minded for your suggestions about better wording.

Reply:
It doesn't have to be tailed with "when creating a GPUTexture", I assumed we'll remove this part in this case.

Reply:
I think it should say, e.g. for maxTextureDimension2D:

`The maximum allowed value for the {{GPUTextureDescriptor/size}}.[=GPUExtent3D/width=} and {{GPUTextureDescriptor/size}}.[=GPUExtent3D/height=] of a [=texture=] with {{GPUTextureDescriptor/dimension}} {{GPUTextureDimension/"2d"}}.`

Reply:
Hmm, I followed Kai's suggestion, with slight revision " ... a [=texture=] created with ..." as Dzmitry suggested.

Reply:
OK either way, but I didn't think it was necessary. The dimension of a texture is always the one it was created with. Also, the texture hasn't been created yet (this validation occurs inside createTexture) so if you want to say "create" it should be
`The maximum allowed value for descriptor.size.width and descriptor.size.height when creating a texture with descriptor.dimension "2d".`
or something but I think that's needlessly verbose.